pkg/downloadhost/domainname.go

```diff
@@ -0,0 +1,43 @@
+package downloadhost
+
+import (
+	"sync"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+const (
+	CLIDownloadsServiceName = "hyperconverged-cluster-cli-download"
+)
+
+type CLIDownloadHost struct {
+	DefaultHost configv1.Hostname
+	CurrentHost configv1.Hostname
+	CRT         string
```

Comment:
tiny nit. i think this name to store the certificate would be a better fit
```suggestion
	Cert         string
```

Reply:
fixed